src/phalanx/services/application.py

```diff
@@ -155,30 +155,42 @@ def lint(self, app_name: str, env_name: str | None) -> bool:
             success &= self._helm.lint_application(app_name, name, values)
         return success
 
-    def lint_all(self) -> bool:
+    def lint_all(self, *, git: bool = False) -> bool:
```

Comment:
It might be a good to use a slightly more self-explanatory argument name for the Python API rather than just adopting the CLI option name (where it does make sense).